src/components/EmptyList.tsx

```diff
@@ -0,0 +1,22 @@
+import { t } from 'i18next'
+import SearchMeta from 'src/types/SearchMeta'
+
+interface IEmptyList {
+  meta?: SearchMeta
+}
+
+export default function EmptyList({ meta }: IEmptyList) {
+  if (!meta) {
+    return null
+  }
+
+  return meta.total === 0 ? (
+    <h3 className="m-4 text-center">{t('ui.noResults')}</h3>
```

Comment:
I'd suggest adding `text-dark  fw-normal` to the classes to make it softer and less header-like.

Reply:
Go with `p-4` instead of margin. Gives it a little more space and is a bit more semantic to use padding here.